go/test/endtoend/cluster/vttablet_process.go

```diff
@@ -168,10 +197,35 @@ func (vttablet *VttabletProcess) TearDown() error {
 	}
 }
 
+// Kill shuts down the running vttablet service
+func (vttablet *VttabletProcess) Kill() error {
+	if vttablet.proc == nil {
+		fmt.Printf("No process found for vttablet %d", vttablet.TabletUID)
+	}
+	if vttablet.proc == nil || vttablet.exit == nil {
+		return nil
+	}
+	// Attempt graceful shutdown with SIGTERM first
+	vttablet.proc.Process.Signal(syscall.SIGTERM)
+
+	os.RemoveAll(vttablet.PidFile)
+
+	select {
+	case err := <-vttablet.exit:
+		vttablet.proc = nil
+		return err
+
+	case <-time.After(10 * time.Second):
+		vttablet.proc.Process.Kill()
+		vttablet.proc = nil
+		return <-vttablet.exit
+	}
+}
```

Comment:
This is almost similar to `TearDown` , we can organise this.

Reply:
Handled it with a parameter